hibernate-core/src/test/java/org/hibernate/orm/test/ondeletecascade/OnDeleteCascadeRemoveTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
We need the LGPL headers here and in all the other files.

Reply:
I'm surprised this didn't fail the build... Is it just that such checks are only present in newer branches?

Reply:
I think the check is not performed for tests classes